recipes/packer_pipeline_node.rb

```diff
@@ -102,7 +102,7 @@
 end
 
 # setup qemu so that we can build images!
-include_recipe 'base::kvm' if node['kernel']['machine'] == 'x86_64'
+include_recipe 'yum-qemu-ev' if node['kernel']['machine'] == 'x86_64'
```

Comment:
Two things:

- [ ] You need to keep ``base::kvm`` but move it AFTER ``yum-qemu-ev``
- [ ] Remove the architecture statement, this should work on ppc64le the same.